alf/environments/parallel_environment.py

```diff
@@ -171,6 +262,8 @@ def close(self):
         logging.info('Closing all processes.')
         for env in self._envs:
             env.close()
+        for env in self._spare_queue:
```

Comment:
clear the promises: _reset_ts, _spare_promises

Reply:
Done.